### PR TITLE
Renamed team.html to members.html

### PR DESCRIPTION
**Pull request by Itzadetunji**

### CODE REVIEW

**Comment by Itzadetunji** [comment]
Abeg do this PR ooo

**Comment by Itzadetunji** [comment]
Converted projects.html to jsx format

File: pages/projects.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+import { Navbar, Footer, Breadcrum } from "../components";
+
+function Projects() {
+  return (
+    <>
+      <Navbar />
+
+      <main className="bg-[#3331C4] pb-12">
+        {/* Change this to the current Active Page */}
+        <Breadcrum currentPage="Sample" />
```
Comment (eni4sure) [inline pages/projects.js]: you forgot to update the breadcrumbs prop

**Review by eni4sure — CHANGES_REQUESTED** [review]

File: pages/projects.js (hunk 1 above)
Comment (Itzadetunji) [reply pages/projects.js]: Done

**Comment by eni4sure** [comment]
Awesome!!

File: pages/projects.js (hunk 1 above)
Comment (Itzadetunji) [reply pages/projects.js]: I forgot to make some commits but I will do them later

**Pull request merged**